monitor/setup.sh

```diff
@@ -116,6 +116,10 @@ elif [ "$RUNNER_OS" = "Linux" ]; then
   echo "REQUESTS_CA_BUNDLE=/home/mitmproxyuser/.mitmproxy/mitmproxy-ca-cert.pem" >> $GITHUB_ENV
   # set environment variable for the Elixir Hex package manager to use the certificate
   echo "HEX_CACERTS_PATH=/home/mitmproxyuser/.mitmproxy/mitmproxy-ca-cert.pem" >> $GITHUB_ENV
+  # copy certificate to Docker directory
+  sudo mkdir -p /etc/docker/certs.d/registry-1.docker.io:443
+  sudo cp ~/mitmproxy-ca-cert.crt /etc/docker/certs.d/registry-1.docker.io:443
```

Comment:
Well I just realized that `registry-1.docker.io:443` might have to be provided as an input value, right? 😕